jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/utils/PayloadCreator.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.homeoffice.digital.sas.jparest.utils;
+
+import java.util.List;
+
+public class PayloadCreator {
+
+    public static String createIDPayload(List<String> fieldNameList, List<String> fieldValueList){
```

Comment:
It would be more robust if we used a hash map here instead of 2 lists as that would keep the two collections of data in sync (key, value)

Reply:
I've moved this to using a map now